lib/linguist/languages.yml

```diff
@@ -4884,6 +4880,14 @@ Objective-J:
   tm_scope: source.js.objj
   ace_mode: text
   language_id: 259
+Octave:
+  type: programming
+  color: "#2299c4"
+  extensions:
+  - ".m"
+  tm_scope: source.octave
+  ace_mode: text
+  language_id: 854759768
```

Comment:
As there is a codemirror mode for Octave, you may as well add it here too.

Reply:
See my previous comment on avoiding codemirror more.

Reply:
Ditto 😉